test/fixtures/applications/rds/my-sql-service/service.yml

```diff
@@ -21,8 +21,9 @@ remote:
 
 local:
   dependencies:
-    - name: mysql
-      version: 5.6.17
+    mysql:
+      type: mysql
```

Comment:
Can we remove this type since its not used? Same for any type that is not exocom / nats.